### PR TITLE
Move Image classification with ConvMixer to keras-core

### PR DESCRIPTION
**Pull request by divyashreepathihalli**
- fixed imports
- removed tfa dependency

### CODE REVIEW

File: examples/keras_io/tensorflow/vision/convmixer.py [hunk 1]
```diff
@@ -0,0 +1,298 @@
...
+"""
+## Final notes
+There's been a recent trend on fusing convolutions with other data-agnostic operations
+like self-attention. Following works are along this line of research:
+* ConViT ([d'Ascoli et al.](https://arxiv.org/abs/2103.10697))
+* CCT ([Hassani et al.](https://arxiv.org/abs/2104.05704))
+* CoAtNet ([Dai et al.](https://arxiv.org/abs/2106.04803))
+| Trained Model | Demo |
+| :--: | :--: |
+| [![Generic badge](https://img.shields.io/badge/🤗%20Model-ConvMixer-black.svg)](https://huggingface.co/keras-io/conv_Mixer) | [![Generic badge](https://img.shields.io/badge/🤗%20Spaces-ConvMixer-black.svg)](https://huggingface.co/spaces/keras-io/conv_Mixer) |
```
Comment (fchollet) [inline examples/keras_io/tensorflow/vision/convmixer.py:297]: You can remove the HF badge it's no longer relevant

**Review by fchollet — COMMENTED** [review]
Thanks for the PR! Does the example work with all backends? If so we should move it out of the tensorflow directory.

File: examples/keras_io/tensorflow/vision/convmixer.py (hunk 1 above)
Comment (divyashreepathihalli) [reply examples/keras_io/tensorflow/vision/convmixer.py]: done

**Comment by divyashreepathihalli** [comment]
> Thanks for the PR! Does the example work with all backends? If so we should move it out of the tensorflow directory.

I had to change data augmentation from a sequential model to a function for it to work with all the backend. I have filed an issue with a repro here - https://github.com/keras-team/keras-io/issues/1540

**Comment by fchollet** [comment]
Keras Core is becoming Keras 3, and we're switching development to the main repository! Please reopen this PR in the [keras-team/keras](https://github.com/keras-team/keras) repository. Unfortunately we aren't able to automatically transfer PRs (but we have transferred all issues).

**Pull request closed without merging**